src/google-drive/authentication.js

```diff
@@ -0,0 +1,81 @@
+let fs = require('fs');
```

Comment:
Please run `npm run lint:fix`. There are a lot of linting errors from semicolons and spacing